core/src/main/resources/hudson/PluginManager/installed_de.properties

```diff
@@ -21,8 +21,8 @@
 # THE SOFTWARE.
 
 No\ plugins\ installed.=Keine Plugins installiert.
-Changes\ will\ take\ effect\ when\ you\ restart\ Jenkins=�nderungen treten nach einem Neustart von Jenkins in Kraft.
-Uncheck\ to\ disable\ the\ plugin=Zum Deaktivieren des Plugins Markierung l�schen
+Changes\ will\ take\ effect\ when\ you\ restart\ Jenkins=\u00C4nderungen treten nach einem Neustart von Jenkins in Kraft.
```

Comment:
According to https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html, property files use `ISO 8859-1` character encoding. This includes German umlauts. I think it is perfectly OK to use them in property files (as we already do in serveral places) because it eases maintenance. The only downside I see is if someone without a German keyboard has to work on these files. But this is not the default maintenance use case.

I propose to use this PR to consequently switch to German umlauts in German localization.